APCAppCore/APCAppCore/Library/Categories/ORKOrderedTask+APCHelper.m

```diff
@@ -105,7 +105,7 @@ + (ORKOrderedTask *)twoFingerTappingIntervalTaskWithIdentifier:(NSString *)ident
         case APCTapHandOptionRight:
             rightHand = YES; break;
         default:
-            rightHand = (((NSUInteger)[NSDate timeIntervalSinceReferenceDate])%2 == 0); break;
+            rightHand = (arc4random()%2 == 0); break;
```

Comment:
arc4random_uniform(2) was actually resulting in "always pick left first" whereas using the modulo method was working. Not sure why.